### PR TITLE
Fix missing migration dependency

### PR DESCRIPTION
**Pull request by neomilium**
This PR contains 2 commits that I wrote few months ago.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -7,6 +7,7 @@ group :test do
   gem 'facter', (ENV['FACTER_GEM_VERSION'] || '~> 2.0'),           require: false
   gem 'puppet-lint-classes_and_types_beginning_with_digits-check', require: false
   gem 'puppet-lint-leading_zero-check',                            require: false
+  gem 'puppet-lint-param-types',                                   require: false
```
Comment (smortex) [inline Gemfile]: The beginning of the file says:

```
# DO NOT EDIT THIS FILE!
# This file is managed by ModuleSync.
```

Can you please move this in a separate PR for opus-codium/modulesync_config?

Comment (neomilium) [reply Gemfile]: Done.

I have no simple way to have `hub` executable right now, so I cant use `./bin/create-pull-requests` script in our `modulesync_config` repo.
Could you made it for me, please?

Thank you.
